lib/Onnxifi/base.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "base.h"
+
+namespace glow {
+namespace onnxifi {
+
+bool Event::signal() {
+    std::lock_guard<std::mutex> guard(mutex_);
+    if (fired_) {
+      return false;
+    }
+    fired_ = true;
+    cond_.notify_all();
+    return true;
+}
+
+void Event::wait() {
+    std::unique_lock<std::mutex> guard(mutex_);
+    cond_.wait(guard, [this] { return fired_; });
+}
+
+onnxStatus Graph::initGraph(const void *onnxModel, size_t onnxModelSize, uint32_t weightCount, const onnxTensorDescriptor *weightDescriptors) {
```

Comment:
80col (maybe clang-format in general)

Reply:
I thought I applied one :) but it does not look right here...

Reply:
apparently master needs some clang-formatting as well, let me submit that first.